modules/pipeline/provider.go

```diff
@@ -25,13 +25,15 @@ import (
 	"github.com/erda-project/erda-infra/providers/httpserver"
 	"github.com/erda-project/erda-proto-go/core/pipeline/cms/pb"
 	_ "github.com/erda-project/erda/modules/pipeline/aop/plugins"
+	"github.com/erda-project/erda/modules/pipeline/providers/trigger"
 )
 
 type provider struct {
 	CmsService         pb.CmsServiceServer `autowired:"erda.core.pipeline.cms.CmsService"`
 	ReconcilerElection election.Interface  `autowired:"etcd-election@reconciler"`
 	GcElection         election.Interface  `autowired:"etcd-election@gc"`
 	Router             httpserver.Router   `autowired:"http-router"`
+	TriggerService     *trigger.TriggerService `autowired:"erda.core.pipeline.trigger.TriggerService"`
```

Comment:
trigger.TriggerService is an interface type, which itself is a pointer

Reply:
done